pyanaconda/modules/storage/initialization.py

```diff
@@ -57,6 +57,7 @@ def enable_installer_mode():
     blivet_flags.ibft = conf.storage.ibft
     blivet_flags.multipath_friendly_names = conf.storage.multipath_friendly_names
     blivet_flags.allow_imperfect_devices = conf.storage.allow_imperfect_devices
+    blivet_flags.btrfs_compression = conf.storage.btrfs_compression
```

Comment:
Please, set `pythonblivetver` to `1:3.3.3-1` in the `anaconda.spec.in` file. @vojtechtrefny, is it the right version?

Reply:
Ah, yeah, I think this is worth bumping up blivet from 3.3.2